plugin.py

```diff
@@ -14,3 +19,25 @@ class LspVolarPlugin(NpmClientHandler):
     package_name = __package__
     server_directory = 'server'
     server_binary_path = os.path.join(server_directory, 'node_modules', '@volar', 'server', 'out', 'index.js')
+
+    @classmethod
+    def is_allowed_to_start(
```

Comment:
I do not like that I had to use is_allowed_to_start.

I think that  `def on_client_configuration_ready(cls, configuration: Dict) -> None:` is a more appropriate hook.
https://github.com/sublimelsp/lsp_utils/blob/master/st3/lsp_utils/generic_client_handler.py#L181

But if I use `on_client_configuration_ready` I face two problems:
1. I have no access to workspace_folders. Alternative solution for that is to use `sublime.active_window().folders()`
2. `cls._server_directory_path()` will return me an empty string when called from `on_client_configuration_ready`. Alternative solution is to use `is_allowed_to_start` and modify the client configuration there.